src/ProjectNeon/Assets/Features/Cards/CardPresenter.cs

```diff
@@ -19,7 +19,7 @@ public void Set(Card card, Action onClick)
     {
         _onClick = onClick;
         _card = card;
-        name.text = _card.name.WithSpaceBetweenWords();
```

Comment:
Why this change? This makes the card less readable, no?

Reply:
@SilasReinagel Sorry, debug code left